kof-operator/internal/k8s/prometheus.go

```diff
@@ -0,0 +1,43 @@
+package k8s
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"log"
+
+	"github.com/k0rdent/kof/kof-operator/internal/models/target"
+	v1 "github.com/prometheus/prometheus/web/api/v1"
+)
+
+const (
+	PrometheusPort     = "9090"
```

Comment:
We need to make this port configurable, in the upcoming changes receiver might be on a different port.